entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityAccessControlHandler.kt

```diff
@@ -394,4 +400,82 @@ class EntityAccessControlHandler(
 
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build<String>()
     }
+
+    @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
+    suspend fun getEntitiesIdsHaveRights(
+        @RequestHeader httpHeaders: HttpHeaders,
+        @RequestParam params: MultiValueMap<String, String>
+    ): ResponseEntity<*> {
+        val count = params.getFirst(QUERY_PARAM_COUNT)?.toBoolean() ?: false
+        val (offset, limit) = extractAndValidatePaginationParameters(
+            params,
+            applicationProperties.pagination.limitDefault,
+            applicationProperties.pagination.limitMax,
+            count
+        )
+        val type = params.getFirst(QUERY_PARAM_TYPE)
+        var q = params.getFirst(QUERY_PARAM_FILTER)
```

Comment:
`var`? really?

Reply:
It's a mistake. I forgot to change this.